src/features/user/types/user-data.ts

```diff
@@ -0,0 +1,10 @@
+export interface  UserData {
+    firstName: string;
+    lastName: string;
+    phone: string;
+    city: string;
+    dateOfBirth: string;
+}
+export interface ResultData extends UserData{
```

Comment:
Do czego to służy?

Reply:
Ten pierwszy interfejs obsługuje typy danych dla zmiennych, które przechowują dane wpisane przez użytkownika. Użytkownik wpisuje wszystkie dane oprócz ID. W tym celu stworzyłem nowy interfejs, który dodatkowo oprócz danych użytkownika obsługuje ID, które jest niezbędne do aktualizacji danych w bazie.